docs-client/src/containers/MethodPage/index.tsx

```diff
@@ -79,14 +79,12 @@ function removeBrackets(headers: string): string {
   return headers.substring(1, length - 1).trim();
 }
 
-function isExactPathMapping(method: Method): boolean {
+function isSingleExactPathMapping(method: Method): boolean {
```

Comment:
We use this value to display different forms in doc service.
If this is `true`, `HttpQueryString` shows up. If not, `EndpointPath` shows up.
When multiple paths are specified, we should display `EndpointPath` to get a whole path from the user.
So this is the right way to achieve the behavior at the moment. But when we implement [this](https://github.com/line/armeria/issues/2167), we should reconsider this behavior. Just left a comment for the remind. 😄